doc/mgmt/SONiC_PIM_SM_HLD.md

```diff
@@ -0,0 +1,1036 @@
+ address                                                                           | Enter RP address in dotted decimal format(A.B.C.D)                |
+   prefix-list                                                                           | Enter the keyword prefix-list then configure with a valid prefix-list-name            |
+   |**Note**         | If the group-address/mask is not specified, the given RP address will map to the default group 224.0.0.0/4.
+| **Default**                                                                   | None  |
+| **Usage Notes**                                                               | PIM-SM specific Rendezous point configuration, .                         |
+| **Command mode**                                                              | CONFIG                                                                    |
+| **Example**                                                                   |                                               |                 
+|                 | a)	To configure a prefix-list to a RP, Sonic(config)# ip pim rp-address 20.1.1.1 prefix-list rp-grp-prefix-list
+| |b)	To configure default group (224.0.0.0/4) for an RP,       Sonic(config)# ip pim rp-address 20.1.1.1    
+|                                                                           | c)	To remove the rp-maddress mapped to a prefix-list , Sonic(config)# no ip pim rp-address 20.1.1.1 prefix-list rp-grp-prefix-list (d) To remove the rp-address along with its group mapping Sonic(config)# no ip pim rp-address 20.1.1.1 |
+**Support**                                                                   | 4.1                                        |
+
+
+
+##### 3.5.2.1.2 ip pim spt-threshold infinity  
+
+
+
+|ip pim spt-threshold infinity  | Command to disable or re-enable shortest path tree (SPT) swtchover for all the multicast groups or a given prefix-list.|  
+| ------------------------------------------------ | ------------------------------------------------------------ |
+|                                  **Syntax**                                   | ip pim [vrf vrf-name] spt-threshold infinity { prefix-list prefix-list-name} |     
+|                                                                               | no ip pim [vrf vrf-name] spt-threshold infinity       |
+| **Parameters**                                                                |                                                                                       |     
+| vrf                                                                           | Applies to specific VRF if VRF input is provided       |     
+|spt-threshold  | Shortest path tree switchover   |
+|infinity       | Disables SPT switchover
+|prefix-list   |  Provide the prefix-list or list of multicast groups on which SPT switchover has to be disabled or re-enabled.
```

Comment:
" or re-enabled" - is this needed? i feel just " Provide the prefix-list or list of multicast groups on which SPT switchover has to be disabled " is fine as no cmd does not take prefix-list

Reply:
Modified